.github/workflows/callable-create-github-container-image-frontend.yml

```
@@ -0,0 +1,79 @@
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.tag || github.ref}}
+
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: npm
+          cache-dependency-path: '${{ inputs.package-dir }}/package-lock.json'
```

actionlint (1.7.4)

property "package-dir" is not defined in object type {service: string; tag: string}

(expression)